examples/Basic usage - Functional.ipynb

```diff
@@ -270,36 +286,78 @@
    "outputs": [],
    "source": [
-    "# Plot df/f0 for all ROIs and trials\n",
+    "# Fetch the colormap object for Cynthia Brewer's Paired color scheme\n",
+    "cmap = plt.get_cmap(\"Paired\")\n",
     "\n",
     "# Get the number of ROIs and trials\n",
-    "n_cell = deltaf.shape[0]\n",
+    "n_roi = result.shape[0]\n",
+    "n_trial = result.shape[1]\n",
+    "\n",
+    "# Find the maximum signal intensities for each ROI\n",
+    "roi_max = [\n",
+    "    np.max([np.max(result[i_roi, i_trial][0]) for i_trial in range(n_trial)])\n",
+    "    for i_roi in range(n_roi)\n",
+    "]\n",
+    "\n",
+    "# Plot our figure using subplot panels\n",
+    "plt.figure(figsize=(16, 10))\n",
+    "for i_roi in range(n_roi):\n",
+    "    for i_trial in range(n_trial):\n",
+    "        # Make subplot axes\n",
+    "        i_subplot = 1 + i_trial * n_roi + i_roi\n",
+    "        plt.subplot(n_trial, n_roi, i_subplot)\n",
+    "        # Plot the data\n",
+    "        plt.plot(\n",
+    "            result[i_roi][i_trial][0, :],\n",
+    "            color=cmap((i_roi * 2 + 1) % cmap.N),\n",
+    "        )\n",
+    "        # Labels and boiler plate\n",
+    "        plt.ylim([-0.05 * roi_max[i_roi], roi_max[i_roi] * 1.05])\n",
+    "        if i_roi == 0:\n",
+    "            plt.ylabel(\"Trial {}\\n\\nSignal intensity\\n(candela per unit area)\".format(i_trial))\n",
+    "        if i_trial == 0:\n",
+    "            plt.title(\"ROI {}\".format(i_roi))\n",
+    "        if i_trial == n_trial - 1:\n",
+    "            plt.xlabel(\"Time (frame number)\")\n",
+    "\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Get the number of ROIs and trials\n",
+    "n_roi = deltaf.shape[0]\n",
     "n_trial = deltaf.shape[1]\n",
     "\n",
     "# Find the maximum df/f0 for each ROI\n",
-    "cell_max = [\n",
-    "    np.max([np.max(deltaf[i_cell, i_trial][0]) for i_trial in range(n_trial)])\n",
-    "    for i_cell in range(n_cell)\n",
+    "roi_max = [\n",
+    "    np.max([np.max(deltaf[i_roi, i_trial][0]) for i_trial in range(n_trial)])\n",
+    "    for i_roi in range(n_roi)\n",
     "]\n",
     "\n",
     "# Plot our figure using subplot panels\n",
     "plt.figure(figsize=(16, 10))\n",
-    "for i_cell in range(n_cell):\n",
+    "for i_roi in range(n_roi):\n",
     "    for i_trial in range(n_trial):\n",
     "        # Make subplot axes\n",
-    "        i_subplot = 1 + i_trial * n_cell + i_cell\n",
-    "        plt.subplot(n_trial, n_cell, i_subplot)\n",
+    "        i_subplot = 1 + i_trial * n_roi + i_roi\n",
+    "        plt.subplot(n_trial, n_roi, i_subplot)\n",
     "        # Plot the data\n",
-    "        n_frames = deltaf[i_cell, i_trial].shape[1]\n",
+    "        n_frames = deltaf[i_roi, i_trial].shape[1]\n",
     "        plt.plot(\n",
     "            np.arange(0, n_frames, dtype=np.float) / sample_frequency,\n",
```

Comment:
Apparently I missed one. Needs to be np.float64 or float to remove deprecation warnings.